config/environments/production.rb

```diff
@@ -38,6 +38,7 @@
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
   # config.force_ssl = true
+  config.ssl_options = { ca_file: ENV['TLS_CA_PATH'] }
```

Comment:
This is for the listening part in rails and as you said this will be handled by `envoy`. We need the CA cert in all the other places we're calling the additional services like RBAC and inventory.

Reply:
That's what I thought 🤔 Is it possible to set up the CA cert for all the other places during runtime? It made the most sense to me to set up the `SSL_CERT_FILE` env variable in `0_clowder-config.rb` but apparently this causes issues with clowdwatch in fedramp

Reply:
Well, we don't really have cloudwatch working :see_no_evil: for now. At least we never had the time to implement it properly.

Reply:
Hmm.. maybe it doesn't matter then 🤷‍♀️  Checking with Dan